libs/ledgerjs/packages/types-cryptoassets/src/index.ts

```diff
@@ -76,7 +203,7 @@ export type ExplorerView = {
 export type CryptoCurrency = CurrencyCommon & {
   type: "CryptoCurrency";
   // unique internal id of a crypto currency
-  id: string;
+  id: CryptoCurrencyId | "LBRY" | "groestcoin";
```

Comment:
why is LRBY and groestcoin not included in the list before ?

Reply:
These are actually typos made in the past. In the object `cryptocurrenciesById` we want the key to be the same as the id.
eg:

```ts
{
   my_coin: {
      id: "my_coin"
   }
}
```

However for these coin the id and the key is different eg key is `groestlcoin` and id is `groestcoin`. To make it compliant with the typing I had to add these values as valid ids even though they are not valid keys